### PR TITLE
etcd-manager - add support for etcd v3.5.3

### PR DESCRIPTION
**Pull request by rifelpet**
https://github.com/etcd-io/etcd/releases/tag/v3.5.3

### CODE REVIEW

File: etcd-manager/pkg/etcdversions/mappings.go [hunk 1]
```diff
@@ -116,7 +118,11 @@ func EtcdVersionForAdoption(fromVersion string) string {
 			return Version_3_4_13
 		}
 	case "3.5":
-		return Version_3_5_1
+		if fromSemver.Patch <= 1 {
+			return Version_3_5_1
+		} else {
+			return Version_3_5_3
+		}
```
Comment (rifelpet) [inline etcd-manager/pkg/etcdversions/mappings.go:125]: I wasn't sure what to do here exactly, whether we want to allow users to keep using 3.5.1 or force them to 3.5.3 (I think that's what this logic handles?)

Comment (hakman) [reply etcd-manager/pkg/etcdversions/mappings.go:125]: I vote for force. This was the reason we don't have 3.5.0 either.

Comment (rifelpet) [reply etcd-manager/pkg/etcdversions/mappings.go]: I've updated this to force 3.5.3 in both functions similar to what we did for 3.5.0 -> 3.5.1

**Review by hakman — APPROVED** [review]

**Comment by justinsb** [comment]
Thanks for doing this @rifelpet !

**Pull request merged**